src/Microsoft.SqlTools.Dmp.Hosting/Localization/sr.xlf

```diff
@@ -0,0 +1,82 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" original="sr.resx" source-language="en">
+    <body>
+      <trans-unit id="CredentialsServiceInvalidCriticalHandle">
+        <source>Invalid CriticalHandle!</source>
+        <target state="new">Invalid CriticalHandle!</target>
+        <note></note>
+      </trans-unit>
+      <trans-unit id="CredentialsServicePasswordLengthExceeded">
```

Comment:
Again please remove credentials service stuff from all the xliff files, and when you do matching cleanup on other projects (taking out the extensibility / hosting messages) remove those from the xliff.